code/computational_geometry/quickhull/Quick_Hull.c

```diff
@@ -0,0 +1,129 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include "bool.h"
+#include "geometry.h"
```

Comment:
Are they a standard library?

Reply:
Nope......I forgot to add them ! 
sorry !